composeApp/src/androidMain/kotlin/io/github/couchtracker/tmdb/TmdbMovie.kt

```diff
@@ -22,12 +22,12 @@ data class TmdbMovie(
     )
 
     suspend fun releaseDates(cache: TmdbCache): List<TmdbReleaseDates> = tmdbGetOrDownload(
-        get = { cache.movieReleaseDatesCacheQueries.get(id, language).awaitAsOneOrNull() },
-        put = { cache.movieReleaseDatesCacheQueries.put(tmdbId = id, language = language, releaseDates = it) },
+        get = { cache.movieReleaseDatesCacheQueries.get(id).awaitAsOneOrNull() },
+        put = { cache.movieReleaseDatesCacheQueries.put(tmdbId = id, releaseDates = it) },
         downloader = {
             it.movies.getDetails(
                 id.value,
-                language.apiParameter,
+                null,
```

Comment:
Why this change? Does the language not change the output of this API?

Reply:
Exactly. We might need to rethink this `TmdbMovie`, since it doesn't make much sense to have this here, but I'm going to leave this to another PR